src/components/Tools/ProgressTracker.vue

```diff
@@ -124,6 +124,22 @@ export default defineComponent({
       }
     },
   },
+  methods: {
+    generateRequirementProgressString(req: string, reqGroup: string): string {
```

Comment:
This name is too long. For example, the prefix `generate` doesn't convey any additional information.